helix-term/src/ui/prompt.rs

```diff
@@ -94,6 +94,10 @@ impl Prompt {
         self
     }
 
+    pub fn prompt(&self) -> Cow<str> {
+        self.prompt.clone()
+    }
```

Comment:
Couldn't this just return `&str`?

Reply:
Resolved by https://github.com/pinelang/helix-tree-explorer/commit/f37c795c96dc84deec65255eb1cb5c964b343c4d